components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/generic/GenericOneTimeTask.java

```diff
@@ -62,8 +62,8 @@ public void notifyLocalTaskRemoval(String taskName) {
         try {
             eventBasedConsumer.destroy();
         } catch (AbstractMethodError e) {
-            logger.warn("Task [" + taskName + "] : Unsupported operation 'destroy()' for Inbound Endpoint: "
-                    + getName() + ". If using a WSO2-released inbound, please upgrade to the latest version. "
+            logger.warn("Task [" + taskName + "] : Unsupported operation 'destroy()' for this version of "
+                    + "PollingConsumer. If using a WSO2-released inbound, please upgrade to the latest version. "
```

Comment:
This should be changed to EventBasedConsumer.

Reply:
Fixed c5ce392640993a899723cc05e1e018822d171360